src/libs/schedule.ts

```diff
@@ -151,20 +133,33 @@ const genericBlocks: Record<
 	}
 };
 
+function isOnLateStartList(date: Date) {
+	const lateStartList = lateStarts as DateList;
+	lateStartList.date.forEach(value => {
```

Comment:
Can you turn this into a `for`..`of` loop?

Reply:
aight